package.json

```diff
@@ -5,13 +5,13 @@
   "main": "src/pages/index.js",
   "private": true,
   "dependencies": {
-    "grommet": "^2.16.3",
-    "grommet-icons": "^4.4.0",
-    "grommet-theme-hpe": "^2.3.0",
-    "next": "^12.0.4",
+    "grommet": "^2.25.1",
+    "grommet-icons": "^4.7.0",
+    "grommet-theme-hpe": "^2.4.0",
```

Comment:
Let's bump up to latest theme if possible -- v3.2.1

Reply:
done!